charts/hivemq-platform/values.yaml

```
@@ -192,6 +192,9 @@ services:
     annotations: {}
     # Labels to add to the service
     labels: {}
+    # Enables PROXY protocol for a HiveMQ listener. Only applicable to MQTT and WebSocket services.
+    # See https://docs.hivemq.com/hivemq/latest/user-guide/proxy-protocol.html
+    # proxyProtocol: false
```

```suggestion
    # hivemqProxyProtocol: false
```

We should make sure this is related to HiveMQ config.